src/Raven.Server/Config/RavenConfiguration.cs

```diff
@@ -500,5 +500,18 @@ public override void Load()
                 }
             }
         }
+
+        public bool DoesKeyExistInSettings(string keyName, bool serverWide = false)
+        {
+            IConfiguration cfg = serverWide ? ServerWideSettings : Settings;
+            
+            if (cfg == null || keyName == null)
+                return false;
+
```

Comment:
Same. It is better just to do it with '.Any`

Reply:
done